src/test/script_tests.cpp

```diff
@@ -145,6 +146,7 @@ CMutableTransaction BuildSpendingTransaction(const CScript& scriptSig, const CMu
 void DoTest(const CScript& scriptPubKey, const CScript& scriptSig, int flags, const std::string& message, int scriptError)
 {
     bool expect = (scriptError == SCRIPT_ERR_OK);
+    int fEnableDIP0020Opcodes = SCRIPT_ENABLE_DIP0020_OPCODES & flags;
```

Comment:
```suggestion
    bool fEnableDIP0020Opcodes = (SCRIPT_ENABLE_DIP0020_OPCODES & flags) != 0;
```

Variables `f` prefixed like this should be `bool` (IMO 😄). Also helps here to avoid implicit `int` -> `bool` conversion few lines below (something we try to avoid in general).

Reply:
Good catch!